projects/extension/src/background/index.ts

```diff
@@ -12,6 +12,39 @@ import {
   ToExtension,
 } from "@substrate/connect-extension-protocol"
 
+const setupStorageBootnodes = () => {
+  let p = localStorage.getItem(polkadot.id)?.split(",")
```

Comment:
localStorage is the browser's storage (not the extension's). I prefer to avoid using chrome.storage.local for this as  chrome.storage APIs are asynchronous while localStorage is sync

Reply:
What's the difference?

Reply:
All mentioned [here](https://developer.chrome.com/docs/extensions/reference/storage/#overview). We could for the sake of consistency use the chrome.localstorage API